ember_debug/object-inspector.js

```diff
@@ -468,8 +468,21 @@ function addProperties(properties, hash) {
       continue;
     }
     let options = { isMandatorySetter: isMandatorySetter(hash, prop) };
+
+    if (typeof hash[prop] === 'object' && hash[prop] !== null) {
+      options.isService = hash[prop].type === 'service';
+    }
+
     if (isComputed(hash[prop])) {
-      options.readOnly = hash[prop]._readOnly;
+        options.dependentKeys = hash[prop]._dependentKeys;
+        if (!options.isService) {
+          try {
+            options.code = hash[prop]._getter.toString();
```

Comment:
Since you always expect this to be a function, could you essentially do `Function.prototype.toString.call(hash[prop]._getter)`?

Reply:
@rwjblue no luck( Tests not  passed